vpn/main.cc

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) 2022 The Brave Authors. All rights reserved.
```

Comment:
I think it's confusing to have main and service_main. I also don't think we want to call this executable "vpn". I think it should be something like `brave_vpn_helper`

Reply:
done, thanks